libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.kt

```diff
@@ -597,19 +604,17 @@ open class SalesforceSDKManager protected constructor(
     }
 
     /**
-     * Optionally enables browser based login instead of web view login. This
-     * should NOT be used directly by apps as this is meant for internal use
-     * based on the value configured on the server.
+     * Optionally enables browser based login instead of web view login.
      *
      * @param browserLoginEnabled True if Chrome should be used for login; false
      * otherwise
      * @param shareBrowserSessionEnabled True enables browser session sharing;
      * false otherwise
      */
     @Synchronized
-    fun setBrowserLoginEnabled(
+    internal fun setBrowserLoginEnabled(
```

Comment:
This is an intentionally breaking change.  I don't think we should have allowed this to begin with.